### ISSUE TITLE
Ensure users can pause or hide animated content

### ISSUE THREAD

**Issue by gilleygoat** [web-infrastructure/doc_architecture#11]
If any part of the content moves, scrolls or blinks for more than 5 seconds, or automatically updates, check that there is a mechanism available to pause, stop or hide the content within 5 seconds.

Animation is allowed if it is essential. Essential here means that if it were removed, it would fundamentally change the information or functionality of the content, and information and functionality cannot be achieved in another way without animation.
## More information

Learn about the ["Pause, Stop, Hide" Success Criterion](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html).
